### PR TITLE
fix failing travis builds

### PR DESCRIPTION
**Pull request by srajangarg**

### CODE REVIEW

**Comment by srajangarg** [comment]
@isuruf

File: .travis.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ matrix:
     - compiler: clang
     - os: osx
   include:
-    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_LATEST_GCC="yes" WITH_COVERAGE="yes" WITH_MPC="yes"
+    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_LATEST_GCC="yes" WITH_MPC="yes"
```
Comment (isuruf) [inline .travis.yml]: Why this change?

Comment (srajangarg) [inline .travis.yml]: Wanted to keep it only 1 job

Comment (srajangarg) [inline .travis.yml]: The longest one, with most options has it

Comment (isuruf) [inline .travis.yml]: Because of this change though, coverage is down.

**Comment by srajangarg** [comment]
@isuruf dropped that commit

**Comment by ranjithkumar007** [comment]
Thanks, @srajangarg.
